Resources/Prototypes/Damage/damage.yml

```diff
@@ -0,0 +1,85 @@
+# Silver: Todo break out into damage_type,damage_class, damage_container yml files when we support loading prototypes by priority.
```

Comment:
This should be solved when using string ids in data fields instead of the prototypes directly (or when prototype references are in).
Relying on the ordering like this while hilarious is going to keep me up at night.